examples/python_django/.gitignore

```diff
@@ -0,0 +1 @@
+key.jwk
```

Comment:
Did we also want to ignore `db.sqlite3`? Contributors may accidentally commit their own updated versions

Reply:
We could, but we don't actually use, it just came with the template. I'll add just to be safe.